runtime/src/lib.rs

```diff
@@ -139,6 +139,8 @@ pub use content;
 pub use content::LimitPerPeriod;
 pub use content::MaxNumber;
 
+pub use argo_bridge;
+
 /// This runtime version.
 #[sp_version::runtime_version]
 pub const VERSION: RuntimeVersion = RuntimeVersion {
```

Comment:
petra should be spec version `2004`, correctly if I am wrong @kdembler

Reply:
@freakstatic please bump